.eslintrc.js

```diff
@@ -49,6 +49,10 @@ module.exports = {
 				"no-mixed-spaces-and-tabs": ["error", "smart-tabs"],
 				"no-shadow": "error",
 				"require-yield": "off",
+				"@typescript-eslint/prefer-nullish-coalescing": [
```

Comment:
I am not really keen on adding "warning" rules. We should either error or not say anything.